3.9/graphs-smart-graphs.md

```diff
@@ -71,6 +71,17 @@ overhead. The more subgraphs are touched the more network cost will apply.
 However the overall performance is never worse than the same query using a
 General Graph.
 
+Benefits of Hybrid SmartGraphs
+-------------------------------
+
+Hybrid SmartGraphs are capable of using Satellite Collections within their graph
+definition. Therefore, defined EdgeDefinitions between SmartCollections and
+SatelliteCollections can be created. As Satellite Collections are globally
+replicated to each participating DB-Server, graph traversals, shortest path,
+weighted traversals and k-shortest-paths queries can be executed partially
+locally on each DB-Server. This means that query execution is guaranteed to be
```

Comment:
```suggestion
locally on each DB-Server. This means a larger part of the query can be executed
```

We should not "guarantee" this here. There may be some code pieces that will not stay local.
e.g. we may introduce an upper limit of Operations per SmartGraph Request to improve LIMIT calls.